rhel7/profiles/stig.profile

```diff
@@ -297,6 +301,9 @@ selections:
     - sysctl_net_ipv4_conf_all_accept_redirects
     - wireless_disable_interfaces
     - mount_option_dev_shm_nodev
+    - mount_option_dev_shm_nodev.severity=low
```

Comment:
I'd say that these severity levels should be changed in the rules themselves rather than in the profile.

Reply:
done